extension/src-language-server/stpa/modelChecking/model-checking.ts

```diff
@@ -162,31 +223,81 @@ const booleanValue = (variable: string, equal: boolean): string => {
  * @returns the LTL string for the given variable.
  */
 const oneValue = (variable: string, value: string, equal: boolean): string => {
-    return variable + (equal ? "==" : "!=") + value;
+    return variable + (equal ? EQ : NEQ) + value;
+};
+/**
+ * A LTL string for a variable with a given enum value.
+ * @param variable The variable to create the LTL string for.
+ * @param value The enum value of the variable.
+ * @returns the LTL string for the given variable.
+ */
+const enumValue = (variable: string, value: string): string => {
+    return variable + EQ + variable + "_Enum." + value;
 };
 
+/**
+ * Determines the operator for the LTL string.
+ * @param equal Determines whether the variable should be (un)equal to the value.
+ * @param inclusive Determines whether the variable should be in the given range or outside of it.
+ * @param first Determines whether the first value is greater or smaller than the second value when equal is true.
+ * @returns 
+ */
+function determineOperator(equal: boolean, inclusive: boolean, first: boolean): string {
+    if (equal && first) {
+        return inclusive ? GTE : GT;
+    } else if (equal && !first){
+        return inclusive ? LTE : LT;
+    } else if (!equal && first) {
+        return inclusive ? LT : LTE;
+    } else {
+        return inclusive ? GT : GTE;
+    }
+}
+
+/**
+ * Operator strings.
+ */
+const GT = ">";
```

Comment:
I would probably prefer these const defintions further at the beginning of the file, before they are used.